docs/api/covidcast-signals/chng.md

```diff
@@ -185,6 +187,24 @@ Note that due to local differences in health record-keeping practices, estimates
 are not always comparable across locations. We are currently working on
 adjustments to correct this spatial bias.
 
+Indicator values for issue dates before 2021-02-21 are merely estimates, as
+these indicators were not yet available in real time. Backfill behavior of
+these estimates is erratic and not indicative of current backfill behavior.
```

Comment:
Do we want to indicate that this problem is being worked on/will eventually be fixed?

Reply:
How about:
```suggestion
these estimates is erratic and not indicative of current backfill behavior. For more
information on this effect and to track updates as we develop a fix, please see
[covidcast-indicators issue #1289: CHNG historical issues are wrong before 2021-02-21](https://github.com/cmu-delphi/covidcast-indicators/issues/1289).
```